### PR TITLE
The Forest Updates

### PR DESCRIPTION
**Pull request by IceOfWraith**
The following have been updated:
-Upgraded to include latest AMP features
-Linux support through Wine/Xvfb and Docker
-Updated existing config settings
-Added missing config settings
-Added metaconfig merging

### CODE REVIEW

**Comment by IceOfWraith** [comment]
The FetchUrl update stage link needs updated before merging.

File: the-forest.kvp [hunk 1]
```diff
@@ -1,35 +1,87 @@
...
-App.CommandLineParameterDelimiter= 
-App.CommandLineParameterFormat=-{0} "{1}"
+Meta.DisplayName=The Forest
+Meta.Description=The Forest Dedicated Server
+Meta.OS=Windows, Linux
+Meta.Arch=x86_64
+Meta.Author=CubeCoders Limited, IceOfWraith
+Meta.URL=https://store.steampowered.com/app/242760/The_Forest/
+Meta.DisplayImageSource=steam:242760
+Meta.EndpointURIFormat=steam://connect/{ip}/{GenericModule.App.ApplicationPort3}
```
Comment (Greelan) [inline the-forest.kvp:8]: Format needs to be fixed - colon and new port format

File: the-forest.kvp [hunk 2]
```diff
@@ -1,35 +1,87 @@
...
+App.RCONHeartbeatMinutes=0
+App.RCONHeartbeatCommand=ping
+App.TelnetLoginFormat={0}
+App.UpdateSources=[{"UpdateStageName": "SteamCMD Download","UpdateSourcePlatform": "All", "UpdateSource": "SteamCMD", "UpdateSourceData": "556450", "ForceDownloadPlatform": "Windows"},{"UpdateStageName": "Base Config Download","UpdateSourcePlatform": "All","UpdateSource": "FetchURL","UpdateSourceData": "https://github.com/IceOfWraith/AMPTemplates/raw/the-forest/the-forestserver.cfg","UpdateSourceTarget": "./556450/"}]
+App.Compatibility=None
+App.SteamUpdateAnonymousLogin=True
+App.SteamForceLoginPrompt=False
+App.RapidStartup=False
+App.MonitorChildProcess=True
+App.MonitorChildProcessWaitMs=1000
```
Comment (Greelan) [inline the-forest.kvp:64]: Might need to be longer

File: the-forest.kvp [hunk 3]
```diff
@@ -1,35 +1,87 @@
...
-Meta.DisplayImageSource=steam:242760
-Meta.DisplayName=The Forest
-Meta.EndpointURIFormat=steam://connect/{ip}/{GenericModule.App.ApplicationPort3}
-Meta.OS=1
-Meta.URL=https://store.steampowered.com/app/242760/The_Forest/
\ No newline at end of file
+Console.AppReadyRegex=^GameServer init success.*$
+Console.UserJoinRegex= 
+Console.UserLeaveRegex= 
+Console.UserChatRegex=
```
Comment (Greelan) [inline the-forest.kvp:76]: Suggest this be "zeroed" with ^$ to avoid unexpected matching of AMP's default regex

File: the-forest.kvp (hunk 2 above)
Comment (IceOfWraith) [reply the-forest.kvp]: Upped it to 3 seconds to be safe.

File: the-forest.kvp (hunk 1 above)
Comment (IceOfWraith) [reply the-forest.kvp]: This has been corrected.

**Review by Greelan — APPROVED** [review]
You’ve picked up on (or responded to) all of my suggestions, including in Discord, so nothing further to add. Looks good to me!

**Pull request merged**